src/logic/addressBook/utils/index.ts

```diff
@@ -172,3 +145,51 @@ export const filterAddressEntries = (
 
     return foundName || foundAddress
   })
+
+export const getEntryIndex = (
+  state: AppReduxState['addressBook'],
+  addressBookEntry: Overwrite<AddressBookEntry, { name?: string }>,
+): number =>
+  state.findIndex(
```

Comment:
This could be just `state.find`, if we never use the index per se.

Reply:
It was originally created inside the reducers where the index is actually used. https://github.com/gnosis/safe-react/blob/9e3a9ddcdd892fdc8ff4bf7976e716b7f475ff3d/src/logic/addressBook/store/reducer/index.ts

Then I extracted it to a commonplace to reuse the lookup logic in the selectors.